corehq/apps/app_manager/models.py

```diff
@@ -2736,6 +2736,14 @@ class AdvancedModule(ModuleBase):
     search_config = SchemaProperty(CaseSearch)
 
     @classmethod
+    def wrap(cls, data):
+        # lazy migration to accommodate search_config as empty list
+        # http://manage.dimagi.com/default.asp?231186
+        if data.get('search_config') == []:
```

Comment:
The UI returns a dictionary, so [it errored](http://manage.dimagi.com/default.asp?231186) when someone tried to set a value. (It could be possible if they edited the app source manually, but that's a brave thing to do. At the moment this feature is only used by UATBC and ICDS. Seems close enough to zero to consider impossible.)